src/pages/index.md

```diff
@@ -3,9 +3,19 @@ template: home-page
 path: /
 title: Home
 hero:
-  title: Privacy Without Borders.
-  body: Secure private data across public blockchains. For defi, interoperability,
-    and the future.
+  title: <span class="no-wrap">Censorship Resistance</span> for DeFi
```

Comment:
The no-wrap thing is interesting 🤔  I wonder if a min-width might not be a better approach, but this should work for now.

Reply:
We're already fiddling with the widths here to make sure there isn't a weird break where "DeFi" ends up all alone on the next line, so we could definitely go this route. In principle, I'm not really a fan of adjusting styles specific to the copy because it seems to defeat the point of the CMS, but it seems like we can't avoid it. I thought maybe using the "no-wrap" class could maybe help with this. 🤷‍♀️